client/src/components/Upload/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Route, BrowserRouter as Router } from 'react-router-dom';
```

Comment:
The router isn't needed here.

<img width="1014" alt="Screenshot 2019-03-24 at 10 32 34" src="https://user-images.githubusercontent.com/25960351/54876513-3e255780-4e22-11e9-8f04-6c1b342c81f6.png">